pom.xml

```diff
@@ -28,11 +28,11 @@
         <module>rest-api-client</module>
         <module>server-core</module>
         <module>local-storage-manager</module>
+        <module>rest-api-cloud-client</module>
         <module>keystore-service</module>
         <module>storage-service</module>
         <!--<module>notify-client</module>-->
         <module>notify-server</module>
-        <module>rest-api-cloud-client</module>
 
```

Comment:
So, I see no conflicts. Yes, indeed, rest-api-cloud-client is mentioned twice, I'll fix it. Module notify-client switched off,, Do we have it implemented? And the last one: we have discused the necessity to replace junit.jupiter with junit.4.12